Alignment/CommonAlignmentProducer/python/ALCARECOTkAlMinBiasHI_cff.py

```diff
@@ -13,4 +13,7 @@
      src = 'hiGeneralTracks'
 )
 
+ALCARECOTkAlMinBiasHI.trackQualities = cms.vstring("highPurity")
+ALCARECOTkAlMinBiasHI.pMin = 3.0 ##GeV
```

Comment:
@jrcastle I just realized that if we need to make the beamspot to run on top of this AlCaReco instead of the usual TkAlMinBias a cut of 3.0 GeV might be too high @mandrenguyen @cerminar